src/restconfig/src/main/java/org/geoserver/rest/security/xml/AuthFilterChainList.java

```diff
@@ -0,0 +1,30 @@
+/* (c) 2025 Open Source Geospatial Foundation - all rights reserved
+ * This code is licensed under the GPL 2.0 license, available at the root
+ * application directory.
+ */
+package org.geoserver.rest.security.xml;
+
+import com.thoughtworks.xstream.annotations.XStreamAlias;
+import com.thoughtworks.xstream.annotations.XStreamImplicit;
+import java.util.ArrayList;
+import java.util.List;
+
+@XStreamAlias("filterChainList")
+public class AuthFilterChainList {
```

Comment:
Same as above, why is it needed? See for example WorspaceController.workspacesGet().